include/radproto/packet.h

```diff
@@ -8,26 +8,30 @@
 #include <string>
 #include <cstdint> //uint8_t, uint32_t
 
-class Packet
+namespace RadProto
 {
-    public:
-        Packet(const std::array<uint8_t, 4096>& m_recvBuffer, size_t bytes, const std::string& secret);
-        Packet(uint8_t type, uint8_t id, const std::array<uint8_t, 16>& auth, const std::vector<Attribute*>& attributes, const std::vector<VendorSpecific*>& vendorSpecific);
-        ~Packet();
-        uint8_t type() const { return m_type; }
-        uint8_t id() const { return m_id; };
-        const std::array<uint8_t, 16>& auth() const { return m_auth; }
-        const std::vector<Attribute*>& attributes() const { return m_attributes; }
-        const std::vector<VendorSpecific*>& vendorSpecific() const { return m_vendorSpecific; }
-        const std::vector<uint8_t> makeSendBuffer(const std::string& secret);
-        Attribute* makeAttribute(uint8_t type, const uint8_t* data, size_t size, const std::string& secret, const std::array<uint8_t, 16>& auth);
+    class Packet
+    {
+        public:
+            Packet(const std::array<uint8_t, 4096>& m_recvBuffer, size_t bytes, const std::string& secret);
```

Comment:
You should not call function arguments `m_<something>`. This scheme is used for class members. Let it be just `buffer`.